setup/system.sh

```diff
@@ -340,10 +340,19 @@ systemctl restart systemd-resolved
 # Configure the Fail2Ban installation to prevent dumb bruce-force attacks against dovecot, postfix, ssh, etc.
 rm -f /etc/fail2ban/jail.local # we used to use this file but don't anymore
 rm -f /etc/fail2ban/jail.d/defaults-debian.conf # removes default config so we can manage all of fail2ban rules in one config
+
+# Check if the user wants to enable Nextcloud, if the user wants it
+# the relevant firejail configuration will be added
+if [ ${DISABLE_NEXTCLOUD} != "0"]; then
```

Comment:
Change to:

if [ "${DISABLE_NEXTCLOUD}" != "0"]; then

or even just:

if [ "${DISABLE_NEXTCLOUD}" ]; then